lib/KModal.vue

```diff
@@ -310,12 +312,35 @@
           // (otherwise KSelect will be trapped inside modal if KSelect is opened a second time)
           if (this.$refs.content.clientHeight !== 0 && !this.containsKSelect) {
             // add a vertical scrollbar if content doesn't fit
-            if (this.$refs.content.scrollHeight > this.$refs.content.clientHeight) {
-              this.$refs.content.style.overflowY = 'auto';
-            }
+            this.$refs.content.style.overflowY =
+              this.$refs.content.scrollHeight > this.$refs.content.clientHeight ? 'auto' : 'hidden';
           }
         }
+        if (this.$refs.content) {
+          this.$refs.content.style.height = 'auto';
```

Comment:
What does the trick here is that we are updating the height of the content to `auto`, and after that we are reading its `scrollHeight`. The problem is that this causes a reflow, and it can cause performance issues considering how often we excecute this method (you can read more about this [here](https://kellegous.com/j/2013/01/26/layout-performance/)).

But this has made me question why do we need to set the height of the content programatically?, I think that a `height: auto` attribute should do the work, which would be basically the same thing we are doing here (setting the heiht to auto and after set its scrollHeight), but without causing reflow.

Let me check with the team if there is a reason why we need to set the height programatically.